draft-ietf-rats-architecture.md

```diff
@@ -476,6 +476,14 @@ to the outside Verifier.
 
 In this situation, the trust model described in {{trustmodel}} is also suitable for this inside Verifier.
 
+## Roles and Entities
```

Comment:
My opinion: I don't like trying to make "Entities" some formal term.   I suspect "implementation" works in most cases in the text below

Reply:
Agree that "Entity" isn't needed in the section heading. But it is helpful to separate the role from the thing that implements the role. Entity is the word most commonly used in the arch to refer to this thing. So it is OK to use it here.

Reply:
There are various uses of entities throughout the document, including composite device. We decided against "device" in favor of "entity" and in the context of this document the use of this term is now well established. We do not want to provide implementation guidance here, but an architectural concept "on how roles can be collapsed" which is important to know to understand for the solution I-Ds because it allows for the omission of internet protocols.

I see that Dave and Ned agree, but this is one of the rare cases in which I am not convinced. I actually think a focus on implementation guidance distracts from the important parts (such as reuse of key material across roles, etc.).

Reply:
When did we "decide against device in favor of entity"?   I don't think I agree with that.